### PR TITLE
Make it easy to change your avatar.

### PR DESCRIPTION
**Pull request by ralphbean**
This makes it so that when you are logged in, anywhere your avatar
appears it will be clickable and will take you to libravatar so you can
change it.  Importantly, you'll be logged into libravatar correctly
which can sometimes be tricky.

### CODE REVIEW

File: pkgdb2/ui/__init__.py [hunk 1]
```diff
@@ -58,14 +58,19 @@ def branches_filter(branches):
...
-    """ Template filter sorting the given branches, Fedora first then EPEL,
-    then whatever is left.
-    """
-    output = '<img class="avatar circle" src="%s"/>' % (
-        pkgdblib.utils.avatar_url(packager, size)
-    )
-
-    return output
+    """ Template filter to produce the libravatar of a given packager. """
+    if flask.g.fas_user and packager == flask.g.fas_user.username:
```
Comment (pypingou) [inline pkgdb2/ui/__init__.py]: There is an is_authenticated utility method that might be good to use here

Comment (ralphbean) [inline pkgdb2/ui/__init__.py]: Good call.

Comment (ralphbean) [inline pkgdb2/ui/__init__.py]: Fixed.

File: pkgdb2/ui/__init__.py [hunk 2]
```diff
@@ -58,14 +58,19 @@ def branches_filter(branches):
...
-    then whatever is left.
-    """
-    output = '<img class="avatar circle" src="%s"/>' % (
-        pkgdblib.utils.avatar_url(packager, size)
-    )
-
-    return output
+    """ Template filter to produce the libravatar of a given packager. """
+    if is_authenticated() and packager == flask.g.fas_user.username:
+        openid_template = 'http://{packager}.id.fedoraproject.org'
```
Comment (puiterwijk) [inline pkgdb2/ui/__init__.py]: Note that the identity includes a trailing slash.

**Comment by puiterwijk** [comment]
Looks good to me

**Comment by pypingou** [comment]
I will add the trailing slash and merge as I would like to cut another release today.

**Pull request merged**